versioned_docs/version-24.10/integrations/data-analytics/sc-datadog-metrics.md

```diff
@@ -43,48 +52,71 @@ apt install centreon-stream-connector-datadog
 </TabItem>
 </Tabs>
 
-## Configuration
+## Configuring your Datadog equipment
+
+You may need to configure your Datadog equipment so that it can receive data from Centreon. Please refer to the documentation of Datadog.
```

Comment:
```suggestion
You may need to configure your Datadog equipment so that it can receive data from Centreon. Please refer to Datadog's documentation.
```